presto-docs/src/main/sphinx/functions/string.rst

```diff
@@ -32,6 +32,10 @@ String Functions
 
     Returns the Unicode code point ``n`` as a single character string.
 
+.. function:: codepoint(varchar) -> n
```

Comment:
Argument names in documentation are names that describe the arguments rather than types. However, the return value is generally the return type. (see all the others in this file)

Write the documentation like this:
```
.. function:: codepoint(string) -> integer

    Returns the Unicode code point of the only character of ``string``.